smartsim/_core/_install/builder.py

```diff
@@ -504,7 +519,9 @@ def build_from_git(self, git_url: str, branch: str, device: str = "cpu") -> None
         self.run_command(build_cmd, cwd=self.rai_build_path)
 
         self._install_backends(device)
-        if self.torch:
+        if self.torch and self.torch_dir:
```

Comment:
would:
```
def user_supplied_backend(backend: str):
  if backend == "torch":
    return self.torch and self.torch_dir
  if backend == "...":
    ...
```
avoid needing comments like the one below?

Reply:
YES!! I should have known when I was adding that comment that this is probably a good block of code to isolate into its own fn 🤦